src/directions/main.ts

```diff
@@ -108,16 +108,17 @@ export default class MapLibreGlDirections extends MapLibreGlDirectionsEvented {
   }
 
   protected async fetch({ method, url, payload }: RequestData) {
-    const response = (await (method === "get"
-      ? await fetch(`${url}?${payload}`, { signal: this.abortController?.signal })
-      : await fetch(`${url}`, {
-          signal: this.abortController?.signal,
-          method: "POST",
-          headers: {
-            "Content-Type": "application/x-www-form-urlencoded",
-          },
-          body: payload,
-        })
+    const response = (await (
+      method === "get"
+        ? await fetch(`${url}?${payload}`, { signal: this.abortController?.signal })
+        : await fetch(`${url}`, {
+            signal: this.abortController?.signal,
+            method: "POST",
+            headers: {
+              "Content-Type": "application/x-www-form-urlencoded",
+            },
+            body: payload,
+          })
```

Comment:
Nothing's really changed here. I guess it just some new formatting rules from the Prettier, after updating its version.